.github/workflows/gh-actions.yml

```diff
@@ -56,6 +56,34 @@ jobs:
       - run: $RUNNER -q --jerry-tests --buildoptions=--compile-flag=-m32,--cpointer-32bit=on
       - run: $RUNNER -q --jerry-tests --buildoptions=--compile-flag=-m32,--cpointer-32bit=on --build-debug
 
+  Win_x86-64_Conformance_Tests_ESNext:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: python $env:RUNNER --test262 update
```

Comment:
As I see we previously wrote simply `$RUNNER`. Why do you write `$env:RUNNER` here and below? Is it intentional. Sorry for the question, but I am not familiar with GH actions too much.

Reply:
Hi, this is `PowerShell`, not `Bash`, the syntax to retrieve `env` is `$env:RUNNER`, blame powershell :(